src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -1765,8 +1765,8 @@ bool
 PAL_JITWriteEnableHolder::JITWriteEnable(bool writeEnable)
 {
     // Use a thread local to track per thread JIT Write enable state
-    // Initialize threads to start with MAP_JIT pages readable and executable (R-X) by default.
-    thread_local bool enabled = (pthread_jit_write_protect_np(1), false);
+    // Per Apple, new threads start with MAP_JIT pages readable and executable (R-X) by default.
```

Comment:
Is there a place in Apple's documentation that we could put a link to for this?

Reply:
Apple said they were going to update `man pthread_jit_write_protect_np` but it is not currently documented.